code/game/objects/items/toys.dm

```diff
@@ -200,6 +277,161 @@
 
 #undef BALLOON_COLORS
 
+/*
+* Balloon animals
+*/
+
+/obj/item/toy/balloon_guy
```

Comment:
I don't know how we can make the earlier code less ugly (yet) but a good first step would be to move all of the toys you've added to a new type, maybe `/obj/item/toy/character_balloon` and then you would make all of these toys subtypes of that type (e.g. `/obj/item/toy/character_balloon/balloon_guy`, `character_balloon/balloon_nukie`) so we can get a consistent framework going with these interactions.